[DOC] Update gRPC compressiong admonition (#4644)
grafana/tempo: docs/sources/tempo/release-notes/v2-7.md

The most important features and enhancements in Tempo 2.7 are highlighted below.

{{< admonition type="note" >}}
This release disables gRPC compression by default to improve performance.

> This change may increase data usage and network traffic, which can impact Grafana Cloud billing, especially for larger clusters. Refer to [gRPC compression](#grpc-compression-disabled) for more information.

Reviewer: ```suggestion
This change will increase data usage and network traffic, which can impact cloud billing depending on your configuration. Refer to [gRPC compression](#grpc-compression-disabled) for more information.
```

This release disables gRPC compression in the querier and distributor for performance reasons. ([#4429](https://github.com/grafana/tempo/pull/4429))
Our benchmark suggests that without compression, queriers and distributors use less CPU and memory.

{{< admonition type="note" >}}

> While disabling gRPC compression improves performance, it may increase data usage and network traffic, which can impact Grafana Cloud billing, especially for larger clusters.

Reviewer: ```suggestion
While disabling gRPC compression improves performance, it will increase data usage and network traffic, which can impact cloud billing depending on your configuration.
```
